tools/esp8266/webApi.cpp

```diff
@@ -261,7 +261,7 @@ void webApi::getLive(JsonObject obj) {
     JsonArray invArr = obj.createNestedArray(F("inverter"));
     obj["refresh_interval"] = SEND_INTERVAL;
 
-    uint8_t list[] = {FLD_UAC, FLD_IAC, FLD_PAC, FLD_F, FLD_PF, FLD_T, FLD_YT, FLD_YD, FLD_PDC, FLD_EFF, FLD_Q, FLD_ALARM_MES_ID};
+    uint8_t list[] = {FLD_UAC, FLD_IAC, FLD_PAC, FLD_F, FLD_PF, FLD_T, FLD_YT, FLD_YD, FLD_PDC, FLD_EFF, FLD_Q};
```

Comment:
Why is the field removed from the list ?

Reply:
Will man den Wert sehen? versteht ihn überhaupt ein Fremder?
Ist für mich kein Visu Wert und per MQTT gibts den weiterhin, wer ihn zum Debuggen sehen möchte, glaube auch im seriellen Log, da dieses ja den MQTT Daten gleicht.
Wenn du ihn wieder haben möchtest: ", FLD_EVT"

Reply:
Wie wäre es, sobald ein Debug Mode eingeschaltet ist, wird dieser zusätzlich Angezeigt?